content/vendor/helm/helm-airgap-builder.md

```diff
@@ -1,17 +1,17 @@
 ---
 date: 2020-01-03
-linktitle: "Helm Airgap Builder"
-title: Helm Airgap builder
+linktitle: "Helm Airgapped Builder"
```

Comment:
This sounds wrong. Makes it like the builder is airgapped. Which is not the case. I think it should be kept as `Airgap` (next line as well)